README.md

```diff
@@ -1,19 +1,40 @@
-
 # ImageLoaderPlugin ![Build Status](https://github.com/ManiVaultStudio/ImageLoaderPlugin/actions/workflows/build.yml/badge.svg?branch=master)
 Image loader plugin for the [ManiVault](https://github.com/ManiVaultStudio/core) framework.
 
 ```bash
 git clone https://github.com/ManiVaultStudio/ImageLoaderPlugin.git
 ```
 
+<p align="middle">
+  <img src="https://github.com/ManiVaultStudio/ImageLoaderPlugin/assets/58806453/bca56677-847a-493a-a1c2-980f2130c136" align="middle"  width="80%" />
+</p>
+
+## Features 
+- Overview of image information for all image files in a given directory:
+  - Size, number of points (pixels), channels and more
+  - Filter images by name
+- Edit meta data like data set and dimension names
+- Select which dimensions are to be loaded
+- Multiple image formats: PNG, JPEG, BPM and (single page) TIF/TIFF
+  - RGB images
+  - "High-dimensional" images, i.e. more than three values per pixel. All grayscale (single channel) image files in a folder are interpreted as part of one single image, e.g. the files in `./data/images/stack/sun/`
+- Data layout interpretation:
+  - Image sequence (default): Each grayscale image in a folder is interpreted as an image channel, i.e., given 100 images of 28x28 pixels this loader creates a dataset of 28x28=784 data points with 100 dimensions (values per pixel). The image dimensions is 28x28 pixel.
+  - Image stack: Each pixel of all grayscale images in a folder are interpreted as an image channel, i.e., given 100 images of 28x28 pixels this loader creates a dataset of 100 data points with 28x28=784 dimensions. The image dimensions remain 28x28 pixel.
+- Apply data conversion:
+  - If a transformation plugin is installed, like the [PointDataConversionPlugin](https://github.com/ManiVaultStudio/PointDataConversionPlugin), you can apply the transformation that it provided upon loading, e.g. `log2` conversions.
+- Subsample images:
+  - Resize an image with linear, bilinear, B-spline, bicubic, Catmull-Rom or Lanczos interpolation filter
+  - Create a image pyramid of user-defined size by downsampling the image on every level 
+
```

Comment:
### Multi-page TIFF
This loader can import layered TIFF images. To also import the dimension name for each image, one can use the user interface as shown below to select the image `TAG` to use for the dimension name:
![image](https://github.com/ManiVaultStudio/ImageLoaderPlugin/assets/12408747/ca6afd23-ccee-464f-a2b8-e009575cb904)